app/code/Magento/PageCache/etc/varnish6.vcl

```diff
@@ -9,13 +9,6 @@ backend default {
     .host = "/* {{ host }} */";
     .port = "/* {{ port }} */";
     .first_byte_timeout = 600s;
-    .probe = {
```

Comment:
This checks need for grace functionality to work, see https://varnish-cache.org/docs/trunk/users-guide/vcl-grace.html